programs/amm/src/curve/curve_type.rs

```diff
@@ -85,4 +88,29 @@ impl CurveType {
                 )
         )
     }
+    /// Get default fee settings
+    pub fn get_default_fee(&self) -> PoolFees {
+        match self {
+            CurveType::ConstantProduct {} => PoolFees {
+                trade_fee_numerator: CONSTANT_PRODUCT_TRADE_FEE_NUMERATOR,
+                trade_fee_denominator: FEE_DENOMINATOR,
+                owner_trade_fee_numerator: CONSTANT_PRODUCT_ADMIN_TRADE_FEE_NUMERATOR,
+                owner_trade_fee_denominator: FEE_DENOMINATOR,
+            },
+            CurveType::Stable { .. } => PoolFees {
+                trade_fee_numerator: STABLE_SWAP_TRADE_FEE_NUMERATOR,
+                trade_fee_denominator: FEE_DENOMINATOR,
+                owner_trade_fee_numerator: STABLE_SWAP_ADMIN_TRADE_FEE_NUMERATOR,
+                owner_trade_fee_denominator: FEE_DENOMINATOR,
+            },
+        }
+    }
+
+    /// Get allowed trade fee bps
+    pub fn get_allowed_trade_fee_bps(&self) -> &[u64] {
+        match self {
+            CurveType::ConstantProduct {} => &[25, 100, 400, 600],
+            CurveType::Stable { .. } => &[1, 4, 10],
```

Comment:
we should include 100

Reply:
Oops. missed for the rust side. Thanks for catching this.

Reply:
Fixed